src/main/resources/script_templates/Deconvolution/DeconWithChirpImg.groovy

```diff
@@ -0,0 +1,69 @@
+// @OpService ops
+// @UIService ui
+// @ConvertService convert
+// @DatasetService data
+
+// Author Chalkie/Brian Northan
+// Prerequisite install http://www.optinav.info/Diffraction-PSF-3D.htm
+
+import ij.IJ;
+import net.imglib2.FinalDimensions
+import net.imglib2.type.numeric.real.FloatType
+import net.imagej.ImgPlus
+import net.imglib2.img.Img
+import net.imglib2.img.display.imagej.ImageJFunctions
+
+def plotProfile() {
+	IJ.resetMinAndMax();
+	IJ.makeLine(0, 32, 511, 32);
+	IJ.run("Plot Profile");
```

Comment:
Semicolons are not required in Groovy. I suggest leaving them away in the example scripts.